test/test_nn.py

```diff
@@ -4707,18 +4707,18 @@ def test_transformerencoderlayer(self):
         ref_output = torch.Tensor([[[2.272644, 0.119035, -0.691669, 0.153486]],
                                    [[2.272644, 0.119035, -0.691669, 0.153486]]])
         self.assertEqual(tuple(result.shape), tuple(ref_output.shape))
-        torch.testing.assert_allclose(result, ref_output)
+        torch.testing.assert_allclose(result, ref_output, rtol=1e-2, atol=1e-5)
```

Comment:
I'm concerned to loose the failure criteria because the original passes the tests.

Reply:
The current `torch.testing.assert_allclose` function asserts that both `rtol` (relative tolerance) and `atol` (absolute tolerance) must be specified. Previously in the numpy version of this test case,  only `atol` was set to `1e-5`.
My changes ensure that this is still maintained, while adding an extra `rtol=1e-2`. This passes the present test case and should not loosen the criteria too much in my opinion.